libraries/rush-lib/src/logic/operations/ProjectLogWritable.ts

```diff
@@ -93,9 +96,16 @@ export class ProjectLogWritable extends TerminalWritable {
     const logPath: string = `${projectFolder}/${relativeLogPath}`;
     const errorLogPath: string = `${projectFolder}/${relativeErrorLogPath}`;
 
+    const { logChunksPath, relativeLogChunksPath } = LogChunksWritable.getLogFilePaths({
```

Comment:
Not following, I adjusted this so the log chunks writable is initialized during `onWriteChunk` and no longer in the constructor.

Reply:
This might be stale. At the time I wrote it there was some weird extra routing going on with an extra splitter instead of `onWriteChunk` writing to two places.

Reply:
Ah, yup I adjusted everything back into `ProjectLogWritable`.